css.css

```diff
@@ -0,0 +1,68 @@
+body {
+    font-family: 'Baloo 2', cursive;
+	margin: 5em;
+}
+
+.collectionContainer1{
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+}
+
+.collectionContainer2{
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+}
+
+.collectionItem{
+    border: solid 0.1em;
+    margin: 0.5em;
+    padding-left: 0.5em;
+    padding-right: 0.5em;
+    padding-bottom: 0.5em;
+}
```

Comment:
Play around with percentage widths at some point if you want to have three items to a row

Reply:
adjusted to 33% and merged into one class